app/web/markdown/handbook.md

```diff
@@ -281,21 +287,21 @@ You can message anyone on the platform. However, please keep our community value
 
 ### How can I leave a reference for someone?
 
-#### After hosting or staying with someone
+#### *After hosting or staying with someone*
```

Comment:
you shouldn't italicize these

Reply:
Will fix. Thanks!

Reply:
Before was better... but current works too